kernelci.org/content/en/docs/contrib.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Contributing Guidelines"
+date: 2022-12-08T09:21:56Z
+draft: false
+weight: 2
+---
+
+KernelCI core project is open for contributions. Contributions may consist of
+adding new build, tests and device types as well as features and bugfixes for
+KernelCI core tools.
+The best way to contribute is to send a PR to [kernelci-core](https://github.com/kernelci/kernelci-core).
+When the PR is created, [KernelCI staging](https://kernelci.org/docs/instances/staging)
+instance takes care of updating the [staging.kernelci.org branch](https://github.com/kernelci/kernelci-core/tree/staging.kernelci.org).
+In general the branch is updated every 8h and a limited set of builds and tests
+is run on it. More detailed information about the logic behind staging runs can
+be found [here](https://kernelci.org/docs/instances/staging).
+
+There are several guidelines which can facilitate the PR review process.
+
+1. Make sure the PR is well described
```

Comment:
We are missing a explanation about KernelCI usage of DCO (Signed-off-by: reviewed-by: tested-by:) in comments.

Reply:
Is it OK to add it in as a follow-up PR?

Reply:
ok

Reply:
@mgalka Maybe create a GitHub issue so we don't forget about this by merging this PR?

Reply:
Have you created an issue for this?

Reply:
@mgalka See above - I haven't found any issue. I think that's the only thing to resolve before it can be merged